component_sdk/python/kfp_component/google/ml_engine/_client.py

```diff
@@ -0,0 +1,208 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import time
+
+import googleapiclient.discovery as discovery
+from googleapiclient import errors
+
+class MLEngineClient:
```

Comment:
Could you please add a TODO list in this client python about what other interfaces will be surfaced. 
For example, model version list as in https://cloud.google.com/ml-engine/reference/rest/

Reply:
I added a module level comment for that.